### PR TITLE
Adding use-herald-action to pull request tools

### PR DESCRIPTION
**Pull request by gagoar**
Hi! I wanted to add `user-herald-action` to your awesome list.  We used it plenty at Coursera and I thought it could be useful for other people as well!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Prow GitHub Actions](https://github.com/jpmcb/prow-github-actions) - Automation of policy enforcement, chat-ops, and automatic PR merging.
 - [Check GitHub Status in your Workflow](https://github.com/crazy-max/ghaction-github-status)
 - [Manage Labels on GitHub (create/rename/update/delete) as Code](https://github.com/crazy-max/ghaction-github-labeler)
+- [Herald rules for Github: Add subscribers, assignees, labels and more to your PR](https://github.com/gagoar/use-herald-action)
```
Comment (gagoar) [inline README.md:146]:
```suggestion
- [Herald rules for GitHub: Add subscribers, assignees, labels and more to your PR](https://github.com/gagoar/use-herald-action)
```

**Review by sdras — CHANGES_REQUESTED** [review]
Please Title-Case as per CONTRIBUTING.md

**Comment by gagoar** [comment]
done!

**Review by sdras — APPROVED** [review]

**Comment by sdras** [comment]
Thanks!

**Pull request merged**